lib/flex/FlexService.js

```diff
@@ -168,31 +168,96 @@ class FlexService {
     });
   }
 
+  /**
+   * Gets all apps and all orgs. If there are no apps and no orgs at the same time, passes error to callback. If entities exist, passes an object with 'app' and 'org' properties.
+   * @param done
+   * @private
+   */
+  _getAllDomainEntities(done) {
+    const result = {};
+
+    async.series([
+      (next) => {
+        this._getDomainEntities(false, (err, data) => {
```

Comment:
This does not feel right. The _getDomainEntities function accepts a boolean parameter and uses it to decide whether to return apps or orgs. I think it is better (for readability purposes) to have getApps and getOrgs (they can still call a common function inside). Alternatively, the argument can be changed to a constant instead of bool.